composer.json

```diff
@@ -14,7 +14,7 @@
         "issues": "https://github.com/acquia/content-hub-php/issues"
     },
     "require": {
-        "acquia/http-hmac-php": "~3.4|~4.1.2",
```

Comment:
We really only need to specify a minimum here. The http-hmac-php spec isn't going to be making future changes that this library won't be compatible with.

I'd suggest instead just changing it to `>=3.4`